abmarl/sim/gridworld/examples/build_grid_example.py

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
Forgot about this... I need to make sure that the grid building from file still works.